client/src/app/core/repositories/polls/poll-repository.service.ts

```diff
@@ -0,0 +1,383 @@
+    amount_global_no: FixedPointNumber;
+    amount_global_abstain: FixedPointNumber;
+}
+
+@Injectable({
+    providedIn: 'root'
+})
+export class PollRepositoryService
+    extends BaseRepositoryWithActiveMeeting<ViewPoll, Poll>
+    implements ModelRequestRepository {
+    public constructor(repoServiceCollector: RepositoryServiceCollector, private optionRepo: OptionRepositoryService) {
+        super(repoServiceCollector, Poll);
+    }
+
+    public getVerboseName = (plural?: boolean): string => {
+        return plural ? 'Polls' : 'Poll';
+    };
+    public getTitle = (viewModel: ViewPoll): string => {
+        return viewModel.title;
+    };
+
+    public getFieldsets(): Fieldsets<Poll> {
+        const listFieldset: (keyof Poll)[] = [
+            'entitled_group_ids',
+            'state',
+            'title',
+            'type',
+            'pollmethod',
+            'content_object_id'
+        ];
+        const detailFieldset: (keyof Poll)[] = listFieldset.concat(
+            'projection_ids',
+            'voted_ids',
+            'votescast',
+            'votesinvalid',
+            'votesvalid',
+            'option_ids',
+            'majority_method',
+            'onehundred_percent_base',
+            'global_option_id'
+        );
+        return {
+            [DEFAULT_FIELDSET]: detailFieldset,
+            list: listFieldset
+        };
+    }
+
+    public getViewModelListByContentObject(contentObject: BaseModel): ViewPoll[] {
+        return this.getViewModelList().filter(_poll => _poll.content_object_id === contentObject.fqid);
+    }
+
+    public getRequestToGetAllModels(): SimplifiedModelRequest {
+        return {
+            viewModelCtor: ViewMeeting,
+            ids: [this.activeMeetingId],
+            follow: [
+                {
+                    idField: 'poll_ids',
+                    fieldset: 'list',
+                    follow: [{ idField: 'content_object_id' }]
+                }
+            ]
+        };
+    }
+
+    public async create(
+        poll: Partial<PollAction.CreateAnalogPollPayload | PollAction.CreateNonAnalogPollPayload>
+    ): Promise<Id> {
+        if (poll.type === PollType.Analog) {
+            return this.createAnalogPoll(poll);
+        } else {
+            return this.createNonAnalogPoll(poll);
+        }
+    }
+
+    public async update(
+        update: Partial<PollAction.CreateAnalogPollPayload | PollAction.CreateNonAnalogPollPayload>,
+        viewPoll: ViewPoll,
+        option?: PollAction.OptionUpdatePayload[]
+    ): Promise<void> {
+        if (update.type === PollType.Analog) {
+            return this.updateAnalogPoll(update, viewPoll, option);
+        } else {
+            return this.updateNonAnalogPoll(update, viewPoll);
+        }
+    }
+
+    private async createAnalogPoll(poll: Partial<PollAction.CreateAnalogPollPayload>): Promise<Id> {
+        const payload: PollAction.CreateAnalogPollPayload = {
+            meeting_id: this.activeMeetingId,
```

Comment:
Ok, we need another solution for this copying :D

I'll create a future issue, so we can care about this, if we have some spare time.